android/src/main/kotlin/com/lahaus/iterable_flutter/IterableFlutterPlugin.kt

```diff
@@ -77,16 +79,25 @@ class IterableFlutterPlugin : FlutterPlugin, MethodCallHandler {
     }
   }
 
-  private fun initialize(apiKey: String, pushIntegrationName: String) {
-    val config = IterableConfig.Builder()
+  private fun initialize(apiKey: String, pushIntegrationName: String, activeLogDebug: Boolean) {
+    val configBuilder = IterableConfig.Builder()
       .setPushIntegrationName(pushIntegrationName)
       .setAutoPushRegistration(false)
-      .setCustomActionHandler { _, _ ->
-        notifyPushNotificationOpened()
-       false
-      }
-      .build()
-    IterableApi.initialize(context, apiKey, config)
+
+    if (activeLogDebug){
+      configBuilder.setLogLevel(Log.DEBUG)
+    }
+
+    setupHandlerPushNotification(configBuilder)
+
+    IterableApi.initialize(context, apiKey, configBuilder.build())
+  }
+
+  private fun setupHandlerPushNotification(configBuilder: IterableConfig.Builder){
+    configBuilder.setCustomActionHandler { _, _ ->
+      notifyPushNotificationOpened()
+      false
+    }
```

Comment:
Super strange, I just want to update the lines since the version that is released has it as true, and the idea is to update and launch another version. :woozy_face: .
But since this method has several reasons to change but better in one method.